### ISSUE TITLE
TWE: Template Issue: Research: Interview Recordings: Transcribe

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#631]
### Overview
We need to create a template issue for `TWE: Research: Interview Recordings: Transcribe` so that we have a consistent, transparent, defined and self-supporting structure for people creating them.

#### Customize this issue
 - Customize this issue by adding the Name of Issue Template. 
    - _Note: An issue template's name is different from the issue title created from the template._
    - [x] Search for `TWE: Research: Interview Transcripts: Clean up and De-identify` by using the browser search (**Mac**: ⌘+F; **PC**: Ctrl+F)
    - [x] Replace the keyword string (including text and the brackets) below with the name of your issue template according to the template naming convention: `TWE: Role: Direct object: Action verb`
        - The role is the primary owner of the issue template (e.g., Research, Design, Product)
        - Direct object should be the item that we're updating or working on (e.g., Wiki, Interviews, Roadmap)
        - Action verb should be the main descriptor of the action items (e.g., Update, Create, Transcribe)

#### Customize the issue settings (located in right sidebar)
Change the issue settings to ensure the issue is properly categorized and easy to manage.
- [x] Under Assignees, assign yourself.
- [x] Under Projects, choose gear, choose Repository, and choose TWE Project Management.
- [x] Add the milestone: Project Management
- [x] Add a feature label specific to what the template is related to, if available.  If not clear what you should use, ask product.

#### Customize template draft
Create a draft of new template, which will be known as our "draft".
- [x] Copy resource 1.01 Content to copy for draft to a new comment on this issue.
- [x] In your draft, fill out the top part (called front matter) with the following information.  See resource 1.02 Front Matter Resources section below for definitions and examples.
    - [x] Name
    - [x] About
    - [x] Title
    - [x] Labels
- [x] Define any Dependencies (or remove the dependency section if there is no dependency)
- [x] Customize the Overview section
- [x] Fill out the Action Items section (in your draft) by defining the steps that are required to be done, keeping the following in mind:
   - [x] The person working on this issue might be new to the org, so the issue should be able to be worked on without any prior knowledge
   - [x] If the person needs to add links, or put documents in specific locations, the place where they can do that action must be linked in the resources section
      - [x] The links in the resources section provided by the issue creator should be in the same order as the Action Items 
      - [x] Refer to resource 1.03 Example Action Items for Finding Wiki Page and Template for an example of how to write clear instructions that avoid compound steps (steps within a step).
   - [x] Number each resource in the order each resource appears in the Action Items
       - [x] Group resources into numbers and decimals (e.g., Resource 1.01, Resource 2.01)
       - [x] Group the resources based on whether they are available when making the issues or they need to be created from this issue
           - If the resource is available, then group it under the **Resources for creating this issue** section (e.g., Resource 1.01, Resource 1.02, Resource 1.03)
           - If the resource will be delivered or created from this issue, then group it under the **Resource links gathered during the making of this issue** section (e.g., Resource 2.01, Resource 2.02, Resource 2.03)
- [x] If documents or folders (sheets, docs, presentations, dashboard, etc.) will be created, find the document template that should be used by checking resource 1.04 WIKI Templates by role pages.  
  - [ ] If there is a template
     - [ ] Add a link to the template into the ` #### Resources for creating this issue`  section of your draft
  - [x] If there is no template yet
    - [x] Put a placeholder into the` #### Resources for creating this issue`  section of the draft `[TWE: Template Issue: [Role]: [Replace ACTION VERB][Replace DIRECT OBJECT]`
- [ ] Get sign-off by Product on the draft

#### If this is for a template draft that needs testing at a later stage, please use the following handoff steps:
- [x] Add `feature: template creation (step 1)` to this issue
- [x] Once the template draft is finished, assign a team member or yourself to create testing issues 
- [x] Once the testings issues are completed, get sign-off by Product on the template draft
- [x] Create or update the template issue on the repo 
- [x] Add the link to the new issue template once created to resource 2.01.  Find by clicking on the **Get started** button to the right of the new template listed on [new issue page](https://github.com/hackforla/internship/issues/new/choose)
- [x] Update the appropriate place in the wiki for your template (see resources section)

#### Resources for creating this issue
- 1.01 Content to copy for draft
```
Draft of template issue for `[Replace NAME OF ISSUE TEMPLATE]`
---
**name:** TWE: Template Issue: [Replace ROLE]: [Replace ACTION VERB][Replace DIRECT OBJECT]
**about:** This issue helps you structure your template issue to be consistent with our other template issues in [Replace DIRECT OBJECT]
** title:** TWE: [Replace ROLE]: [Replace DIRECT OBJECT]: [Replace ACTION VERB]
**labels:** 'feature: board / google drive / tools, feature: missing, milestone: missing, role: missing, size: missing', 'Participant Type: missing', 'complexity: missing'
**assignees:**

---
#### Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- Customize the issue
    - [ ] Copy and paste entire template text in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
    - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
    - [ ] Search for `[Replace`
    - [ ] Replace with whatever the keyword or information is asking for.
- Customize the issue settings (located in right sidebar)
    - [ ] Under Labels, add labels for any labels identified as missing
    - [ ] Remove the corresponding missing labels
    - [ ] Under Projects, choose gear, choose "Repository," and choose TWE: Project Board.
    - [ ] Add the milestone: [Replace MILESTONE]
    - [ ] Add a feature label specific to what the template is related to, if available. If not clear what you should use, ask product.
- Customize dependencies 
    - [ ] If no dependencies, delete dependency section
    - [ ] If dependencies, add the dependencies (any issue or reason that would prevent the action items from being completed)
- Remove this section.
    - [ ] Delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
We need to [Replace ACTION VERB] [Replace DIRECT OBJECT] for [Replace REASON].

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, coding, etc.

### Resources
#### Resources for creating this issue
- 1.01
- 1.02

#### Resources gathered during the completion of this issue
- 2.01
- 2.02
```

- 1.02 Front Matter Resources
   <details>
   <summary>Front Matter Fields and Meaning</summary>

   - name: (name that appear on the new issue tab)
   - about: (is the description that appears next to the issue template on the new issue tab)
   - title: (the title that appears when the issue is created.  User then customizes this title for their issue)
       - Use the naming convention: TWE: Role: Direct object: Action verb
       - If more than one word should go into the direct object spot, use multiple square brackets with the keyword (see example below)
   - labels: (choose the labels that will be the same each time, and choose missing labels for the ones that depend on content)
   - assignees:  
   </details>
   <details>
   <summary>Front Matter Field Example</summary>

   - name: 'TWE: Template Issue: Research: Create Roadmap'
   - about:  This issue helps you structure your template issue to be consistent with our other template issues in roadmap
   - title: 'TWE: Research: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF RESEARCH METHOD]: Roadmap : Create'
   - labels: 'feature: research', 'feature: roadmap', 'milestone: missing', 'role: UI/UX research', 'Participant Type: missing', 'Research 000: missing', 'complexity: missing'
   - 'size: 0.50pt'
   - assignees: ''
   </details>

- 1.03 Example Action Items for Finding Wiki Page and Template
   <details>
   <summary>Example Action Items</summary>

   ```
   ### Action Items
   - Customize wiki page
        - [ ] Go to link 1.01 Internship Wiki, Join the team page
        - [ ] Find wiki page for this issue by looking at role pages.
        - [ ] Copy the link for the wiki page.
        - [ ] Add the link to Resource 2.01
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   - Customize document template.
       - [ ] Go to link 1.02 Design Process Overview
       - [ ] Find document template that should be used in this issue.
       - [ ] Copy the link for the template.
       - [ ] Add the link to Resource 2.02
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   ```

   </details>

   <details>
   <summary>Example Resource Lists</summary>

   ```
   ### Resources for creating this issue
   - 1.01. [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team)
   - 1.02. [Design Process Overview](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

   ### Resource links gathered during the making of this issue
   - 2.01. [Wiki page for [Replace TYPE OF ROLE]]
   - 2.02. [TWE: Template: [Replace NAME OF TEMPLATE]]
   ```

   </details>

- 1.04 WIKI Templates by role pages
   - [TWE: WIKI: UX Research Guides Templates](https://github.com/hackforla/internship/wiki/UX-Research-Guides-Templates) 
   - [Design](https://github.com/hackforla/internship/wiki/Design-Process-Overview)
   - [Product](https://github.com/hackforla/internship/wiki/Product)
   - [Business Analyst / Data Science]
   - [Software Engineer]
   - [Marketing]
   - default document templates, in case you don't have one yet, can be found on the [Design Process Overview page](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

#### Resources gathered during the completion of this issue
- 2.01 [TWE: Research: Interview Recording: Transcribe](https://github.com/hackforla/internship/issues/new?assignees=&labels=complexity%3A+small%2C+feature%3A+recording+transcription%2C+feature%3A+research%2C+milestone%3A+missing%2C+Participant+Type%3A+missing%2C+Research+000%3A+missing%2C+role%3A+UI%2FUX+research%2C+size%3A+0.25pt&projects=&template=research--transcribe-user-interview-recordings.md&title=TWE%3A+IS%5BReplace+YY%5D%3A+RP%5BReplace+000%5D%3A+%5BReplace+TYPE+OF+PARTICIPANT%5D+%5BReplace+PARTICIPANT+NUMBER%5D+%5BReplace+TYPE+OF+INTERVIEW%5D+Interview+Recording%3A+Transcribe)

**Comment by pandanista** [comment]
# Draft of template issue for Transcribing User Interview Recordings and Preparing for De-identification  

**name:** Research: Transcribe User Interview Recordings and Prepare for De-identifying transcripts 
**about:** The process to transcribe interview recordings and get ready to de-identify transcripts
**title:** TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings: Set Up and Transcribe
**labels:** 'feature: recording transcription', 'feature: research', 'Participant Type: missing', 'Research 000: missing', 'size: 1pt', 'role: UI/UX research', 'milestone: missing'
**assignees:** 

---
#### Instructions for creating this issue
The person who creates this issue should use links in the section "Resources for creating this issue" (i.e., 1.01, 1.02, etc.) to customize this issue.
- Customize the issue
    - [ ] Open the "Research Overview" wiki page (Resource 1.01)
    - [ ] Choose the appropriate research roadmap or research plan folder (ie., RP###) on the wiki page 
    - Use the information on the wiki page and in research roadmap or research plan to customize this issue
    - [ ] Copy and paste entire template in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
    - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
       - Year 
           - [ ] Find/Replace `[Replace YY]` with research year (Two digit format, ex: "22" for year "2022") 
       - Research Plan Number
            - [ ] Find/Replace `[Replace 000]` with the research plan number (Three digit format, ex: "001" for research plan "RP001") 
       - Type of Participant
           - [ ] Find/Replace `[Replace TYPE OF PARTICIPANT]` with the participant type (ex: "Intern" or "Mentor")  
       - Type of Interview
           - [ ] Find/Replace `[Replace TYPE OF INTERVIEW]` with the type of interview (ex: "Intake" or "Exit")
    - [ ] Copy and paste the edited template from text editor back into this Github Issue, replacing all original text
    - [ ] Re-check all the checkboxes above after copying and pasting the edited template
    - [ ] Close the text editor
    - Update the issue title
        - [ ] Copy 
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings: Transcribe 
        ``` 
         - [ ] Choose "Edit" in the title section
        - [ ] Paste the text you copied into the issue title
        - [ ] Choose "Save"
- Customize the issue settings (located in right sidebar)
    - [ ] Under Assignees, assign a Researcher if it has been determined. Otherwise, leave it blank.
    - [ ] Under Labels,  choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Research: RP[Replace 000]`
    - [ ] Under Labels, choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Participant Type: [Replace TYPE OF PARTICIPANT]`
    - [ ] Remove the `Research 000: missing` and `Participant Type: missing` labels
    - [ ] Under Projects, choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), choose `Repository`, and choose `Program Management`
    - [ ] Add the milestone: choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and choose `Research Analysis`
    - [ ] Remove the label `milestone: missing`
- Customize dependencies 
    - [ ] If you know of dependencies, add them below (a dependency is any issue or reason that would prevent the action items from being completed)
- Remove this section
    - [ ] After all the checkboxes above are complete and the issue is ready to be worked on, delete all text below the "Draft of template issue..." (top heading) and above "Overview" (heading below) -- i.e., the issue assignee will never see this part

### Dependencies  
 - [ ] Org rep has purchased a premium version of Otter.ai
 - [ ] The assignee is added to 1Password's Internship UI/UX vault
 - [ ] Interview recording upload form needs to be created 
 - [ ] Interview recordings have been uploaded and moved into Internship - PII's "My Drive"

### Overview
We need to transcribe RP[Replace 000]'s [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] interview recordings so that we can move on to de-identify the transcripts. 

During these tasks you will be asked to add link to the Resources/Instructions section, and that section is at the bottom of this issue (go view it now).

Another tip is to use split browser windows side by side when working on any GitHub issue, so you can edit the issue on one window, and check the resources or click through links on the other window to avoid a lot of back and forth. Make sure to refresh both windows to reflect any changes you made, including updating the issue or checking the boxes on one of them.


### Action Items
- [ ] UX lead asks admin to add the team member working on this issue to the 1Password internship-UX's vault so that they can have access to Otter.ai in case they need it.
- [ ]  UX lead adds the assignee to the Internship - PII's "My Drive" as a Content Manager so that they have access to internship - PII's My Drive where the recordings and transcriptions with PII are stored.
   - [ ] UX lead accesses the Internship - PII's "My Drive"
   - [ ] Choose "Manage members", which is located towards the top right side of the browser <img width="200" alt="Screenshot 2023-08-09 at 6 09 36 PM" src="https://github.com/hackforla/internship/assets/57029070/f7cc7fbe-1b00-4ded-8b44-b10fdbccae46">
   - [ ] Enter the assignee's email address and select the role as "Content Manager" 
   - [ ] Confirm with the assignee that they have access to the Internship - PII's "My Drive" 

#### Customize Resource Links
- Customize Resource for Wiki Page Link
    - [ ] Go to the wiki page: Research Output Overview (Resources # 1.01)
    - [ ] Find relevant wiki page for RP[Replace 000]
    - [ ] Copy the link for the wiki page
    - [ ] In Resources # 2.01, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked <img width="100" alt="Screenshot 2023-02-21 at 6 47 15 PM" src="https://user-images.githubusercontent.com/57029070/220508425-5d0c0a1b-ef87-4ef2-a683-d4a21d5b2672.png">

- Customize Resource for this Research Plan's Google Drive Folder
    - [ ] Open the Google Drive's Research by Type Folder (Resources # 1.02)
    - [ ] Choose the `[Replace TYPE OF PARTICIPANT]` folder
    - [ ] Copy the link of the `[Replace TYPE OF PARTICIPANT]` folder
        1. Choose the three vertical dots on the right side of the `[Replace TYPE OF PARTICIPANT]` folder  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [ ] In Resources # 2.02, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

- Customize Resource for Interview Recordings and Transcriptions Tracking Sheet 
    - [ ] Choose "TWE: Template: Interview Recordings and Transcriptions Tracking Sheet" (Resources # 1.03)
    - [ ] Choose "USE TEMPLATE" to make a copy of the document
    - Rename and customize the file by following the steps
        - [ ] Copy 
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings and Transcriptions Tracking Sheet
        ```
        - [ ] Choose "File" > "Rename", and paste the text you copied into the top field    
       <img width="150" alt="Screenshot 2023-01-30 at 1 35 41 PM" src="https://user-images.githubusercontent.com/57029070/215600780-d8004229-17a3-409a-9d91-e4af9c904824.png"> <img width="700" alt="Screenshot 2023-01-30 at 1 34 45 PM" src="https://user-images.githubusercontent.com/57029070/215601015-cefd546c-8496-4b49-8643-ccfae3e4f9ca.png">
        - [ ] The name of the file is now updated
        - [ ] Copy 
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings and Transcriptions Tracking Sheet
        ```
        - [ ] Choose the header row (row 1)
        - [ ] Choose the formula bar, and paste the text you copied into it <img width="800" alt="Screenshot 2023-01-30 at 1 44 02 PM" src="https://user-images.githubusercontent.com/57029070/215602453-b2f08ba0-156b-4424-be5a-35b1f68b2be8.png">
        - [ ] The title of the spreadsheet is now updated
        - [ ] Copy 
        ```
        [Replace TYPE OF PARTICIPANT] Number
        ```
        - [ ] Choose cell A2
        - [ ] Choose the formula bar, paste the text you copied <img width="360" alt="Screenshot 2023-01-30 at 1 48 32 PM" src="https://user-images.githubusercontent.com/57029070/215602930-c3933344-d5a4-46df-85ed-3557e8bc6d17.png">
        - [ ] The cell of the spreadsheet is now updated
    - [ ] Save the spreadsheet into this Research Plan's Google Drive folder in the Internship's shared drive (Resource # 2.02)
        1.  Choose File > Move (or Move icon)
        1.  Choose "All locations"
        1.  Choose Shared drives > Internship > Internships > Research > Research by Type > [Replace TYPE OF PARTICIPANT] > folder for RP[Replace 000]
        1.  Choose "Move"
        1.  Choose "Move" when it asks if you want to change ownership to a shared drive
        1.  Dismiss the notification that says the file in now in the shared drive
    - [ ] Choose "Share"
    - [ ] Choose "Copy link" and "Done"
    - [ ] In Resources # 2.03, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

 - Customize Resource for De-identified Participants List Spreadsheet in the Shared Google Drive to Keep Track of Any Stakeholders Mentioned in Interview Sessions
    - [ ] Choose "TWE: Template: De-identified Participants List from Interviews (UXR Excluded)" (Resources # 1.04)
    - [ ] Choose "USE TEMPLATE" to make a copy of the document
    - Rename and customize the file by following the steps
        - [ ] Copy 
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: De-identified Participants List from Interviews (UXR Excluded)
        ```
        - [ ] Choose "File" > "Rename", and paste the text you copied into the top field <img width="500" alt="Screenshot 2024-09-25 at 5 52 33 PM" src="https://github.com/user-attachments/assets/936978a5-df0e-47d2-8d7e-30922929f1f2">
        - [ ] The name of the file is now updated
        - [ ] Copy 
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: De-identified Participants List from Interviews (UXR Excluded)
        ```
        - [ ] Choose the header row (row 1)
        - [ ] Choose the formula bar, and paste the text you copied into it <img width="500" alt="Screenshot 2024-09-25 at 4 46 08 PM" src="https://github.com/user-attachments/assets/75f92c89-ecc6-4d37-8e1a-8026464f427a">
        - [ ] The title of the spreadsheet is now updated
    - [ ] Move the file to RP[Replace 000]'s folder in the shared Google Drive
        1. Choose File > Move > All Locations > Shared drives > Internship > Internships > Research > Research by Plan # and Name > RP[Replace 000] > Move
        1.  Choose "Move" when it asks if you want to change ownership to a shared drive
        1.  Dismiss the notification that says the file in now in the shared drive
    - [ ] Choose "Share" on the top right corner of the spreadsheet
    - [ ] Choose "Copy link" and "Done"
    - [ ] In Resources # 2.04, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

- Customize Resource for Interview Recording Folder stored in the Internship - PII's `My Drive`
    - [ ] Make sure you are logged into your Google account associated with TWE project so you will be able to access the Internship - PII's 'My Drive' in the next steps 
    - [ ] Choose Shared drives > Internship - PII > Internship - PII
    - [ ] Choose `TWE Interview Recordings by Plan #` folder in Internship - PII's `My drive` (Resources # 1.05) 
    - [ ] Locate the video recording folder for RP[Replace 000] inside the folder `TWE Interview Recordings by Plan #`
    - [ ] Copy the link of the video recording folder for RP[Replace 000] 
        1. Choose the three vertical dots on the right side of the folder  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [ ] In Resources # 2.05, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

- Customize Resource for Participants List Spreadsheet to Save in the Video Recording Folder for RP[Replace 000] in the Internship - PII's `My Drive`
    - [ ] Choose "TWE: Template: Participants List from Interviews (UXR Excluded)" (Resources # 1.06)
    - [ ] Choose "USE TEMPLATE" to make a copy of the document
    - Rename and customize the file by following the steps
        - [ ] Copy 
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: Participants List from Interviews (UXR Excluded)
        ```
        - [ ] Choose "File" > "Rename", and paste the text you copied into the top field <img width="500" alt="Screenshot 2024-09-25 at 6 07 00 PM" src="https://github.com/user-attachments/assets/de115851-2be3-4dcb-b69d-11eb44a48715">
        - [ ] The name of the file is now updated
        - [ ] Copy 
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: Participants List from Interviews (UXR Excluded)
        ```
        - [ ] Choose the header row (row 1)
        - [ ] Choose the formula bar, and paste the text you copied into it <img width="500" alt="Screenshot 2024-09-25 at 5 33 30 PM" src="https://github.com/user-attachments/assets/366e9f87-cf0f-40c6-a840-12824ec2f3f1">
        - [ ] The title of the spreadsheet is now updated
    - [ ] Save the spreadsheet into RP[Replace 000] Interview Recording Folder in the Internship's PII drive (Resource # 2.04)
        1.  Choose File > Move (or Move icon)
        1.  Choose "All locations"
        1.  Choose Shared drives > Internship - PII > Internship - PII > TWE Interview Recordings by Plan # > folder for RP[Replace 000]
        1.  Choose "Move"
        1.  Choose "Move" when it asks if you want to change ownership to a shared drive
        1.  Dismiss the notification that says the file in now in the shared drive
    - [ ] Choose "Share" on the top right corner of the spreadsheet
    - [ ] Choose "Copy link" and "Done"
    - [ ] In Resources # 2.06, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

#### Rename Uploaded Recording Video Files If Needed 
- [ ] Open the folder linked in Resource # 2.05
- [ ] Make sure each recording video file in the folder is named as `RP[Replace 000]-UX Researcher Abbreviation###-[Replace TYPE OF PARTICIPANT] Abbreviation###`
    - If the recording video files are not in the format above, please rename the recording files in the format above first before taking any more actions
    - An example looks like this: `RP006-U007-I001.txt`, which means this is an interview transcript for research plan 006, conducted by UX Researcher 007, and the interviewee is Intern 001
    - In case if you would like to double check to ensure the accuracy, both UX Researcher Abbreviation### and [Replace TYPE OF PARTICIPANT] Abbreviation### can be found in `TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Roll Call and Session Table` in RP[Replace 000] folder linked in Resources # 2.02

#### Transcribe Interviews with Otter.ai when transcripts are not available
- [ ] Open `RP[Replace 000] Interview Recording Folder` linked in Resource # 2.05
- [ ] Check each interview recording file folder to see if there are transcripts available
    - [ ] If yes, move on to `#### Update Interview Recordings and Transcriptions Tracking Sheet` section
    - [ ] If no, do the following to transcribe the video files
        - [ ] Download each recording that needs to be transcribed from the Google Drive's recording folder to a temporary folder locally
        - [ ] Log into 1password (Resources # 1.07)
            1. Choose internship-UI/UX vault
            1. Choose <img width="150" alt="Screenshot 2023-10-12 at 3 43 57 PM" src="https://github.com/hackforla/internship/assets/57029070/7469b511-fac3-4fc1-9de3-b13d6407c662"> to see login info for Otter.ai account
        - [ ] Access Otter.ai (Resource # 1.08) by copying and pasting the login info from 1password vault
        - [ ] Confirm you are on a paid plan at the time of transcribing (Otter.ai only allows 3 uploads for the free tier lifetime)
            - [ ] If you are not on a paid plan, let the UX Lead know about it by making an update as a comment in this issue
        - Complete the following steps using Otter.ai:
            - [ ] Choose "Import" on the upper right corner of the Otter.ai interface once logged in
            - [ ] Upload the video file, and let Otter.ai transcribe the file for you
            - [ ] Once finished, download the .txt file
            - [ ] Rename each .txt transcript file you downloaded in `RP[Replace 000]-UX Researcher Abbreviation###-[Replace TYPE OF PARTICIPANT] Abbreviation###` format
                - An example looks like this: `RP006-U007-I001.txt`, which means this is an interview transcript for research plan 006, conducted by UX Researcher 007, and the interviewee is Intern 001
                - Please double-check the UX Researcher number and participant number in `TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Roll Call and Session Table` in RP[Replace 000] folder in Resources # 2.02
            - [ ] Upload the .txt file into the corresponding recording folder (Resources # 2.05)
                - So there will be two types of files named in the `RP[Replace 000]-UX Researcher Abbreviation###-[Replace TYPE OF PARTICIPANT] Abbreviation###` format. One is a .mp4 video from the interview while the other one is a transcript in .txt.
        - [ ] Delete the local temporary folder with recording files from your own device, such as your laptop
    
#### Update Interview Recordings and Transcriptions Tracking Sheet
- [ ] Open `TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Roll Call and Session Table` inside `Google Drive Folder: RP[Replace 000]` (Resources # 2.02)
- [ ] Identify the highest number of `[Replace TYPE OF PARTICIPANT]` in the "Session Table" tab of `TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Roll Call and Session Table`
- [ ] Open the Interview Recordings and Transcriptions Tracking Sheet you just created (Resources # 2.03) 
    - [ ] Double check if the title in the header is the same as the file name. If not, please edit the file name the same as the header.
- [ ] In Column A of the Tracking Sheet, enter all the participant numbers. 
     - Delete or add rows to match the highest number of participants found in the "Session Table" tab of `TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Roll Call and Session Table`.
- [ ] Open `RP[Replace 000] Interview Recording Folder` linked in Resource # 2.05
- [ ] Count the total number of recording files (.MP4 files) to see if the number matches with the highest number of participants in the session table or if there is any recording file missing
    - [ ] If the numbers match, that means there is no missing recording file
    - [ ] If the numbers don't match, then there might be missing recording file(s)
        - Check the "Session Table" tab of `Interview Roll Call and Session Table` to see if the missing interview was conducted or not
        - Check both Internship's Google Drive and Internship - PII's My Drive to locate the missing interview recording file(s)
        - Get in touch with the UX Researcher(s) who conducted the interview (listed in the Interview Roll Call and Session Table in RP[Replace 000]'s Google Drive folder) to retrieve the recording files if needed
        - Move the missing recording file(s) to the correct folder once located
        - Rename the recording file(s) in the format as `RP[Replace 000]-UX Researcher Abbreviation###-[Replace TYPE OF PARTICIPANT] Abbreviation###` 
        - If the missing files can't be located, please notify the UX Lead in the Tracking Sheet by inserting a comment in the cell where the Row of `[Replace TYPE OF PARTICIPANT] Number` and the Column B `Recording Link` intersect
- [ ] Open the session table tab in [TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Roll Call and Session Table](https://docs.google.com/spreadsheets/d/1dTlIOhtZ5fCNo0GrfKHlOTbdvsRlPUaLIfHk8Cc7Zrw/edit?usp=sharing) inside `RP[Replace 000] Folder`(Resources # 2.02)
    - [ ] Locate the [Replace TYPE OF PARTICIPANT] number in Column C, "[Replace TYPE OF PARTICIPANT] #" 
    - [ ] Copy the corresponding recording link in Column J, "Recording link"
- [ ] Paste the link of the recording video file into the matching participant's cell in Column B `Recording Link` of the Tracking Sheet (Resources # 2.03) 
- [ ] Locate .txt files in the corresponding recording folder (Resources # 2.05)
- [ ] Copy the link(s) of the .txt files 
    1. Choose the three vertical dots on the right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
    1. Choose "Share"
    1. Choose "Get link"
    1. Choose "Copy Link" and "Done"
- [ ] Paste each transcript file URL in the matching participant's cell in Column C `Transcription Link` of the Tracking Sheet (Resources # 2.03)
- [ ] Make sure all the recording video and transcript links are copied and pasted into all matching participants' cells, leaving the cells with no matching recording videos blank

#### Set Up A Folder for De-identified Transcripts in the RP[Replace 000] folder (Resources # 2.02)
- [ ] Go to RP[Replace 000] folder (Resources # 2.02)
- [ ] Create a new folder for the de-identified transcripts in the RP[Replace 000] folder by choosing "New" and "New folder" on the left side bar  <img width="127" alt="Screenshot 2023-01-27 at 2 34 43 PM" src="https://user-images.githubusercontent.com/57029070/215219196-2e8b1712-6c74-4893-a1aa-7db51ca48e19.png">   <img width="200" alt="Screenshot 2023-01-27 at 2 35 00 PM" src="https://user-images.githubusercontent.com/57029070/215219378-052eefb3-64e5-447f-941d-1c2d04f3f1f2.png">
- [ ] Name the "New folder" by copying 
```
RP[Replace 000] De-identified Transcripts
```
- [ ] Paste the text you copied into the folder name
- [ ] Copy the link of the folder
    1. Choose the three vertical dots on the right side of the folder <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
    1. Choose "Share"
    1. Choose "Get link"
    1. Choose "Copy Link" and "Done"
- [ ] Paste the copied link into parentheses at the end of the line in Resources # 2.07
- [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

#### Update the wiki page for this research plan (Resources # 2.01)
- [ ] Open the link in Resource 2.01
- [ ] Find the "Interview Recordings & Transcripts" section
- [ ] Copy and paste Resource 2.07 (both the text and URL) into the "Interview Recordings & Transcripts" section
- [ ] Choose "Save Page"
- [ ] Review with UX Lead 
- [ ] The assignee checks in with UXR Lead to create new issues for de-identifying the transcripts
- [ ] UXR Lead or PM removes the assignee from the Internship - PII's "My Drive" when closing the issue if the assignee does not need to access the Internship - PII's "My Drive" any longer
- [ ] Product sign-off


### Resources/Instructions
#### Resources for creating this issue
1.01 [Wiki: Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
1.02 [Google Drive Folder: Research by Type](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=share_link)
1.03 [TWE: Template: Interview Recordings and Transcriptions Tracking Sheet](https://docs.google.com/spreadsheets/d/1aeUFd5tZQoJCriTzJc2L-L0KS7sXmwhveewb1dviX3I/template/preview)
1.04 [TWE: Template: De-identified Participants List from Interviews (UXR Excluded)](https://docs.google.com/spreadsheets/d/19-jrpj8RxFzAfk09ToxR1H6xNrok_1bgmNfH00CtkxM/template/preview)
1.05* [TWE Interview Recordings by Plan #](https://drive.google.com/drive/folders/1lvP_k4qGKD05QzbHBtIpEl4B3X7ylzoh?usp=sharing)
1.06 [TWE: Template: Participants List from Interviews (UXR Excluded)](https://docs.google.com/spreadsheets/d/1rVk63jdWqjF8sIayo1N4bbfRsCfLUd8YPr1dEdfq6K4/template/preview)
1.07 [1password](https://start.1password.com/signin?l=en)
1.08 [Otter.ai](https://www.otter.ai/)


#### Resources gathered during the completion of this issue
2.01 [Wiki: Research Plan: RP[Replace 000]]
2.02 [Google Drive Folder: RP[Replace 000]
2.03 [TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings and Transcriptions Tracking Sheet]
2.04 [TWE: IS[Replace YY]: RP[Replace 000]: De-identified Participants List from Interviews (UXR Excluded)]
2.05* [RP[Replace 000] Interview Recording Folder]
2.06* [TWE: IS[Replace YY]: RP[Replace 000]: Participants List from Interviews (UXR Excluded)]
2.07 [RP[Replace 000] De-identified Transcripts]

* indicates this folder can only be accessed from the Internship - PII's "My Drive"

**Comment by pandanista** [comment]
[This template](https://github.com/hackforla/internship/issues/335#issuecomment-1760447927) has been updated by removing the `renaming recording files`, `transcribing the recordings`, and `updating the recording and transcript tracking sheet` sections. However, I decided to keep the `Set Up A Folder for De-identified Transcripts in the RP[Replace 000] folder`, `Update the Wiki Page`, and `Review and Hand-off` sections in this template as they are necessary steps to the set-up phase of the transcription.

- [The transcribing template](https://github.com/hackforla/internship/issues/631#issuecomment-2420152100) has been updated. The issue template draft is step 1 and step 2(b)
    - We will incorporate more changes from testing, thus it stays in the review column so we can update the template as needed
- Testing issue has been created in #632, which is Step 2a;
- Tester will create a test, which is Step 3.

**Comment by pandanista** [comment]
- [The transcribe template draft](https://github.com/hackforla/internship/issues/631#issuecomment-2420152100) has been tested, and both tests went well.
    - #632 
    - #634 
- [The transcribe template draft](https://github.com/hackforla/internship/issues/631#issuecomment-2420152100) has been revised to remove the renaming file section since it was incorporated into the set up phase template.
- Closing this issue now since the template has been deployed.  
    - [TWE: Research: Interview Recording: Transcribe](https://github.com/hackforla/internship/issues/new?assignees=&labels=complexity%3A+small%2C+feature%3A+recording+transcription%2C+feature%3A+research%2C+milestone%3A+missing%2C+Participant+Type%3A+missing%2C+Research+000%3A+missing%2C+role%3A+UI%2FUX+research%2C+size%3A+0.25pt&projects=&template=research--transcribe-user-interview-recordings.md&title=TWE%3A+IS%5BReplace+YY%5D%3A+RP%5BReplace+000%5D%3A+%5BReplace+TYPE+OF+PARTICIPANT%5D+%5BReplace+PARTICIPANT+NUMBER%5D+%5BReplace+TYPE+OF+INTERVIEW%5D+Interview+Recording%3A+Transcribe)

Tagging @sunannie27 and @KC-skc for visibility.
